ci-operator/jobs/devfile/integration-tests/devfile-integration-tests-main-periodics.yaml

```diff
@@ -1,7 +1,7 @@
 periodics:
 - agent: kubernetes
   cluster: build05
-  cron: 12 0 3,19 * *
+  cron: 0 2 * * 5
```

Comment:
Use old cron configuration for console

Reply:
fixed in https://github.com/openshift/release/pull/53140/commits/41c10053c310963005662e32116fe5af9a32637b